extensions/positron-r/package.nls.json

```diff
@@ -34,6 +34,7 @@
 	"r.configuration.title-dev": "Advanced",
 	"r.configuration.customRootFolders.markdownDescription": "List of additional folders to search for R installations. These folders are searched after and in the same way as the default folder for your operating system (e.g. `C:/Program Files/R` on Windows).",
 	"r.configuration.customBinaries.markdownDescription": "List of additional R binaries. If you want to use an R installation that is not automatically discovered, provide the path to its binary here. For example, on Windows this might look like `C:/some/unusual/location/R-4.4.1/bin/x64/R.exe`.",
+	"r.configuration.interpreters.exclude.markdownDescription": "List of R interpreter paths or folders (absolute paths) to exclude from the available R installations. These interpreters will not be displayed in the Positron UI.\n\nExample: On Windows, add `C:/some/unusual/location/R-4.4.1/bin/x64/R.exe` to exclude the specific interpreter, or `C:/some/unusual/location` to exclude any interpreters within the directory.\n\nRequires a restart to take effect.",
```

Comment:
Two observations that might lead to re-wording:

* "List of R interpreter paths or folders (absolute paths)...": I know (well, suspect) this is trying to cover both specific binary paths or HQ/root folders, but I think it leaves room for confusion. Maybe borrow concrete wording from the inclusion settings?
* The example: I suspect this setting will be used almost exclusively for Workbench, in which case an example based on some realistic linux server scenario might turn on more light bulbs than a Windows desktop example.

Reply:
I like it!